sound/soc/sof/hw-spi.c

```diff
@@ -167,20 +167,6 @@ static irqreturn_t spi_irq_handler(int irq __maybe_unused, void *context)
 	return IRQ_NONE;
 }
 
-static irqreturn_t spi_irq_thread(int irq __maybe_unused, void *context __maybe_unused)
-{
-	// read SPI data into local buffer and determine IPC cmd or reply
-
-	/*
-	 * if reply. Handle Immediate reply from DSP Core and set DSP
-	 * state to ready
-	 */
-
-	/* if cmd, Handle messages from DSP Core */
-
-	return IRQ_HANDLED;
-}
-
```

Comment:
@lyakh can you do me a favor and not bundle things aren't ready for upstream with things that are. Specifically Haswell/SKL/KBL/Compressed/SPI are not shared upstream just yet, so it helps if your patches follow this split.

Reply:
sorry, what branch should I be making patches against? I was using topic/sof-dev, should I use integration/... instead?

Reply:
@lyakh we should use topic/sof-dev but if you look at the patches posted upstream HSW/KBL/SKL/Compressed/SPI are not included in them. So it will be easier to maintain the squash/rebase cycles if you split for ex. your commit: "ASoC: SOF: eliminate redundant mailbox reads" into 2. One intended for the files in the upstream patches and another one for those not in it (hsw.c etc).

Reply:
I see, but how should I know which ones are and which ones aren't included? I think we need a separate branch, really.

Reply:
@lyakh check topic/sof-dev-rebase, everything after 'add build support' is not upstream yet. Ideally your patches should apply cleanly on the initial batch. Thanks!

Reply:
@plbossart let me take a semi-qualified guess: is this the commit, that I should be branching from?
> commit af23c43d6a9148df393735a9fe3c8a45e8a6435d
> Author: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
> Date:   Mon May 21 18:19:44 2018 -0500
> 
>     ASoC: Intel: boards: initial support for Hifiberry Digiplus w/ WM8804